### ISSUE TITLE
No exchange rate being returned after 2017-11-22

### ISSUE THREAD

**Issue by KanchanChauhan** [alexprengere/currencyconverter#5]
Looks like APIs are not working after the date `2017-11-22.` Can you please look into it. Below is the screenshot. Thanks.

<img width="1011" alt="screen shot 2018-02-18 at 5 04 42 pm" src="https://user-images.githubusercontent.com/16913064/36351318-fb077e62-14cd-11e8-9856-1143a2917645.png">

**Comment by alexprengere** [comment]
Hello,
You seem to have an old version of the currency converter. Note that the data is embedded within the package, to avoid making expensive HTTP calls to an external service when requesting every single rate. So, an old package means old data. To always use up to date data, you have two solutions:

* update your currency converter version, either from PyPI, or from GitHub (updated more often)
* at initialization time, you can instantiate the converter like this (documented at the end of the [README](https://github.com/alexprengere/currencyconverter/#other-attributes)):
  ```python
  from currency_converter import CurrencyConverter
  c = CurrencyConverter('http://www.ecb.int/stats/eurofxref/eurofxref-hist.zip')
  ```
  This way, the latest data will be fetched when creating the `c` object, you just have to have access to the internet at initialization time.

**Comment by alexprengere** [comment]
And since it was quite some time since I pushed to PyPI, I just released version 0.13.3
